docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -24,10 +24,38 @@ Also requires <<privileges-list-indices,manage privileges>> on all indices that
 [[put-search-application-path-params]]
 ==== {api-path-parms-title}
 
+
 `create`::
 (Optional, Boolean) If `true`, this request cannot replace or update existing Search Applications.
 Defaults to `false`.
 
+`<body>`::
+(Optional, object)
+Contains parameters for a search application:
++
+.Properties of `<body>` objects
+[%collapsible%open]
+====
+`indices`::
+(Required, array of strings)
+The <<indices,indices>> associated with this search application. All indices need to exist in order to be added to a search application.
+
+`analytics_collection_name`::
```

Comment:
I think we should have removed this param, at least from the requests / responses given that we're not using it - and it's confusing for users to have something that has no practical purpose.

Reply:
You are absolutely correct - I didn't want to remove it until @aznick confirmed, but he did last night and I'm removing it now 😁